frontend/libs/studio-components/src/components/StudioBetaTag/StudioBetaTag.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { StudioBetaTag, type StudioBetaTagProps } from './StudioBetaTag';
+import { testRootClassNameAppending } from '../../test-utils/testRootClassNameAppending';
+
+describe('StudioBetaTag', () => {
+  it('should render the "Beta" text', () => {
+    renderStudioBetaTag();
+    expect(screen.getByText('Beta')).toBeInTheDocument();
+  });
+
+  it('Appends given classname to internal classname when no image is given', () => {
```

Comment:
Is the `it` a copy paste trap? I don't understand the  context of the text `when no image is given`. 😊 
```suggestion
  it('Appends given classname to internal classname when no image is given', () => {
```

Reply:
hehehehhe, fixed now